webrecorder/wacz-uploader
# Additional files after initial selection

Allow users to choose additional files after the initial selection. Resolves https://github.com/webrecorder/wacz-uploader/issues/14

Also added includes some small dev server improvements that I'll comment on, can be pulled out into a separate PR if that's easier.

### Manual testing
1. Run app with `npm run dev`
2. Choose files with drag/drop or click
3. Click "Add more files" and select files. Verify files are shown in file list
4. Try to add a file that's already been added. Verify file list only contains one instance of the file
5. Try removing and adding a file. Verify file list updates as expected

### Screenshots
<img width="473" alt="Screen Shot 2022-11-09 at 10 14 33 AM" src="https://user-images.githubusercontent.com/4672952/200883438-01278b5f-7270-4a51-895b-e7de614dc5e0.png">

#### Opinions
UI design: I went with the quickest option for selecting more files so drag/drop isn't enabled in this view (cc: @Shrinks99)
